string frame
string detect_type
string group_name
string name
geometry_msgs/Point position
geometry_msgs/Quaternion quaternion

string FRAME_WORLD=usbcam_world
string FRAME_HEAD=usbcam_head
string FRAME_ROBOT=robot

string DETECT_TYPE_ARUCO=aruco
string DETECT_TYPE_COLOR=color
string DETECT_TYPE_POINTCLOUD=pointcloud

string NAME_GOAL=goal
string NAME_TOOL=tool
